Description: remove git usage in gemspec
Forwarded: no
Author: Cédric Boutillier <boutil@debian.org>
Last-Update: 2021-10-28

--- a/bindex.gemspec
+++ b/bindex.gemspec
@@ -14,7 +14,7 @@
 
   spec.required_ruby_version = ">= 2.0.0"
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir.glob('**/*').reject { |f| f.start_with?('debian/') }
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
